src/Security/Member.php

```diff
@@ -1205,7 +1205,7 @@ public static function mapInCMSGroups($groups = null)
             return ArrayList::create()->map();
         }
 
-        if (count($groups ?? []) == 0) {
+        if (count($groups) === 0) {
```

Comment:
Leaving this behaviour alone - just tidying up a little (`$groups` is never not countable here, and `===` is more correct)